### ISSUE TITLE
[DOC] Lacking information about heudiconv commands populate-templates etc

### ISSUE THREAD

**Issue by askieslinger** [nipy/heudiconv#696]
### Summary

Where can I find more information about the heudiconv commands mentioned [here ](https://heudiconv.readthedocs.io/en/latest/usage.html)? 

> <html><body>

<html><body>

--command | Possible choices: heuristics, heuristic-info, ls, populate-templates, sanitize-jsons, treat-jsons, populate-intended-for Custom action to be performed on provided files instead of regular operation.
-- | --

</body>
</html>

</body>
</html>

Specific information about each workflow would help a lot to be able to understand and use the tool.

**Comment by yarikoptic** [comment]
yeah, we should improve API (make them proper sub-commands) and documentation on this. ATM it is just looking at the code and searching through docs to find more description and examples of use as e.g. for `populate-templates`:
https://heudiconv.readthedocs.io/en/latest/search.html?q=populate-templates&check_keywords=yes&area=default would lead you to
- https://heudiconv.readthedocs.io/en/latest/api/bids.html?highlight=populate-templates#heudiconv.bids.populate_bids_templates stating "Premake BIDS text files with templates" and
- https://heudiconv.readthedocs.io/en/latest/usage.html?highlight=populate-templates#batch-jobs would show example of use of it .
